code/modules/clothing/suits/marine_armor/ert.dm

```diff
@@ -79,6 +79,7 @@
 	item_state = "armor"
 	item_state_slots = null
 	contained_sprite = TRUE
+	slowdown = SLOWDOWN_ARMOR_LIGHT
```

Comment:
I would like to add this armor only covers the CHEST . And has pretty much the same or less stats than a jacket .  Also you are adding the slowdown variable wrong